vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -468,7 +468,9 @@ - (void)listOneTimeTips:(void (NS_NOESCAPE ^)(NSArray<BATPublisherInfo *> *))com
 
 - (void)tipPublisherDirectly:(BATPublisherInfo *)publisher amount:(int)amount currency:(NSString *)currency
 {
-  ledger->DoDirectTip(std::string(publisher.id.UTF8String), amount, std::string(currency.UTF8String));
+  ledger->DoDirectTip(std::string(publisher.id.UTF8String), amount, std::string(currency.UTF8String), ^(bool unverified){
+    // TODO(cg505): handle unverified tip
```

Comment:
Is there anyway to know if a verified publisher tip failed due to insufficient balance? (assuming the user's local balance state wasn't updated and when updated wouldn't of allowed them to tip in the first place)